pyproject.toml

```diff
@@ -0,0 +1,43 @@
+[build-system]
+requires = ["setuptools>=41", "wheel", "setuptools-git-versioning"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name="fmiprecice"
+dynamic = [ "version" ]
+dependencies = [
+    "fmpy", "numpy", "pyprecice>=3.0"
+]
+requires-python = ">=3.8"
+authors = [
+    { name = "The preCICE Developers",  email="info@precice.org"}
```

Comment:
Do we want to list Leonard Willeke explicitly as an author?

Reply:
I see this mainly as what we want moving forward. I think it's fine like this. We have citation guidelines in the documentation.